src/apps/inventory/models.py

```diff
@@ -47,3 +47,14 @@ def save(self, *args, **kwargs):
 
     def get_absolute_url(self):
         return reverse("product_detail", args=[str(self.category.slug), str(self.slug)])
+
+
+class ProductImage(models.Model):
+    product = models.ForeignKey(Product, on_delete=models.CASCADE, related_name="images")
+    image = models.ImageField(upload_to="product_images/")
+    is_active = models.BooleanField(default=True)
```

Comment:
How are you using this is_active field, what does it mean, we have the same flag in product model are they related?